.env.sample

```diff
@@ -119,4 +119,7 @@ CRONTAB_SCHEDULE =
 
 
 # Change this to set the max default weeks to allow. Default is currently 16. The issue is the end dates in Canvas currently are set 10 years out so it can't calculate the range.
-MAX_DEFAULT_WEEKS = 
\ No newline at end of file
+MAX_DEFAULT_WEEKS = 
+
+# How long to cache some URL's on the client (Defualt 3600 seconds)
```

Comment:
what do think our configuration option will be? based on the cron update schedule?

Reply:
Ideally yes, it should calculate when the cron run will next occur and use that. 

However we don't always set the RUN_AT_TIMES value on this server (though we could) and that value would be something like 2:00AM. So it would have to be like

`
# If RUN_AT_TIMES is defined calculate how long we should cache for
if RUN_AT_TIMES:
  # Do some calculation to subtract the next time cron will run from the current time and set that as the cache time. Maybe add a 15 minutes for padding since cron takes awhile.
  # TODO Implement code that does this  ;)
`

Reply:
I don't think this would be very many lines of code but not feels like it would require more testing than just setting a fixed value, so could create a new issue.